cmd/voyager/app/options/options.go

```diff
@@ -32,4 +34,6 @@ func (s *Config) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVarP(&s.LoadbalancerImageName, "haproxy-image", "h", s.LoadbalancerImageName, "haproxy image name to be run")
 
 	fs.StringVar(&s.IngressClass, "ingress-class", "", "Ingress class handled by voyager. Unset by default. Set to voyager to only handle ingress with annotation kubernetes.io/ingress.class=voyager.")
+
+	fs.BoolVar(&s.EnableAnalytics, "analytics", true, "Send analytical event to Google Analytics")
```

Comment:
Use EnableAnalytics here instead of true directly.
```
fs.BoolVar(&s.EnableAnalytics, "analytics", EnableAnalytics, "Send analytical event to Google Analytics")
```